cscs-checks/system/io/ior_check.py

```diff
@@ -1,160 +1,165 @@
 import os
+import getpass
 
 import reframe as rfm
 import reframe.utility.sanity as sn
 
+fs = {
+    '/scratch/snx1600tds': {
+        'valid_systems': ['dom:gpu'],
+        'dom': {
+            'num_tasks': 2,
+            # 1 task per node to avoid cache effects on read (other options
+            # like -C did produce the desired impact) 8 tasks are enough
+            # to get ~peak perf (write 5.4 GB/s, read 4.3 GB/s)
+        }
+    },
+    '/scratch/snx1600': {
+        'valid_systems': ['daint:gpu'],
+        'daint': {
+            'num_tasks': 1,  # to validate
+        }
+    },
+    '/scratch/snx3000tds': {
+        'valid_systems': ['dom:gpu'],
+        'dom': {
+            'num_tasks': 2,  # to validate
+        }
+    },
+    '/scratch/snx3000': {
+        'valid_systems': ['daint:gpu'],
+        'daint': {
+            'num_tasks': 1,  # to validate
+        }
+    },
+    '/users': {
+        'valid_systems': ['daint:gpu', 'dom:gpu', 'fulen:normal'],
+        'ior_block_size': '8g',
+        'daint': {
+            'num_tasks': 1,  # to validate
+        },
+        'dom': {
+            'num_tasks': 1,  # to validate
+        },
+        'fulen': {
+            'num_tasks': 1,
+            'build_system_cc': 'mpicc',
+            'build_system_cxx': 'mpic++',
```

Comment:
These should be set in the PrgEnv configuration for Fulen.

Reply:
@victorusu `PrgEnv-gnu` compilers are set to `cc` and `CC` on Fulen currently. Is that correct?